prince/README.md

```diff
@@ -121,3 +121,48 @@ GPU name: Tesla V100-SXM2-32GB-LS
 CPU matmul elapsed: 1.1984939575195312 sec.
 GPU matmul elapsed: 0.01778721809387207 sec.
 ```
+
+# Mujoco Installation on Prince
+
+This tutorial assumes that you have already installec conda in your scratch folder. Please note that your conda environment must be stored in $SCRATCH not in $HOME directory. You will get weird buildlock errors otherwise !
```

Comment:
```suggestion
This tutorial assumes that you have already installed conda in your scratch folder. Please note that your conda environment must be stored in $SCRATCH not in $HOME directory. You will get weird buildlock errors otherwise !
```